awx/ui_next/src/screens/Setting/shared/ShareFields.test.jsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
```

Comment:
This file is supposed to be named "SharedFields" I think

Reply:
😄 I think you're right.